airbyte-webapp/src/components/connection/CatalogTree/next/DestinationNamespaceModal.tsx

```diff
@@ -0,0 +1,346 @@
+import { Field, FieldProps, Form, Formik } from "formik";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import * as yup from "yup";
+
+import { Button } from "components/ui/Button";
+import { Input } from "components/ui/Input";
+import { ModalBody, ModalFooter } from "components/ui/Modal";
+import { Text } from "components/ui/Text";
+
+import { NamespaceDefinitionType } from "../../../../core/request/AirbyteClient";
+import { FormikConnectionFormValues } from "../../../../views/Connection/ConnectionForm/formConfig";
+import LabeledRadioButton from "../../../LabeledRadioButton";
+import styles from "./DestinationNamespaceModal.module.scss";
+
+interface ExampleSettingsTableProps {
+  columns: Array<{ id: string; displayName: string }>;
+  data: Array<Record<string, string>>;
+}
+
+const ExampleSettingsTable: React.FC<ExampleSettingsTableProps> = ({ columns, data }) => {
```

Comment:
This should be extracted to its own file along with the example table data generation. It would make this file easier to maintain